Source/FieldSolver/WarpX_QED_Field_Pushers.cpp

```diff
@@ -98,14 +105,19 @@ WarpX::Hybrid_QED_Push (int lev, PatchType patch_type, Real a_dt)
     {
         Real wt = amrex::second();
 
-        // Get boxes for E and B
-        const Box& tbx  = mfi.tilebox(Bx_nodal_flag);
-        const Box& tby  = mfi.tilebox(By_nodal_flag);
-        const Box& tbz  = mfi.tilebox(Bz_nodal_flag);
+        // Get boxes for E, B, and J
+
+        const Box& tbx = mfi.tilebox(Bx->ixType().ixType());
```

Comment:
Just curius:
what is `Bx->ixType().ixType()` doing? Is this a typo?
And are the vars `Bx_nodal_flag`, ... still used elsewhere?

Reply:
- The first `ixType` is a method of the AMReX class `Box` that returns an object of class `IndexType` (https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1Box.html). The second `ixType` is a method of the AMReX class `IndexType` that returns an object of class `IntVect` (https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1IndexType.html).
_On the AMReX side, it could maybe make more sense if the method `ixType` of the class `IndexType`, that returns an `IntVect`, was named differently or if one could hide the conversion from `IndexType` to `IntVect` in a new method of the `Box` class directly._
- Regarding flags like `Bx_nodal_flag`, you can have a look at the open issue #797.

Reply:
Thanks!